dsl/docker-multiarch/images/python.groovy

```diff
@@ -34,10 +34,12 @@ for v in */; do
 	docker build -t "$repo:$v" "$v"
 	docker build -t "$repo:$v-onbuild" "$v/onbuild"
 	docker build -t "$repo:$v-slim" "$v/slim"
+	docker build -t "$repo:$v-alpine" "$v/alpine"
```

Comment:
This is good, but we'll also need to make sure this doesn't try on arches which Alpine doesn't support (ie, those outside https://github.com/tianon/jenkins-groovy/blob/b11361e40e555e23f0d64be118b25433ca1d18b7/vars/multiarch.groovy#L39-L46).

If you'd like, I'm happy to take over from here, but how I'd like to implement this (for now) is to have this do something similar to `busybox` (https://github.com/tianon/jenkins-groovy/blob/b11361e40e555e23f0d64be118b25433ca1d18b7/dsl/docker-multiarch/images/busybox.groovy#L77-L87) and check whether the `FROM` image exists, and only build/tag if it does (perhaps by removing the directory entirely in a loop before this one when it doesn't, and then it could be more generic like `busybox`'s is).